darts/models/forecasting/regression_ensemble_model.py

```diff
@@ -172,9 +195,11 @@ def fit(
         # prepare the forecasting models for further predicting by fitting them with the entire data
 
         # Some models (incl. Neural-Network based models) may need to be 'reset' to allow being retrained from scratch
-        self.models = [model.untrained_model() for model in self.models]
+        self.forecasting_models = [
```

Comment:
we need to return here before when retrain=False
```
if not self.retrain_forecasting_models:
    return self
```

Reply:
I decided to update the `training_series` attribute of the models to make the behavior of `predict()` more intuitive (otherwise, there might be discrepancies between the forecasting models and the regressor, especially in the prediction time indexes)